opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -66,7 +67,7 @@ namespace Opm {
             AllProperties = IntProperties | DoubleProperties
         };
 
-        EclipseState(std::shared_ptr< const Deck > deck , const ParseContext& parseContext);
+        EclipseState(std::shared_ptr< const Deck > deck , const ParseContext parseContext = ParseContext());
```

Comment:
The default argument is redundant when ParseContext is a member - it would be default constructed anyway. In this case it's better to offer two constructors.